memcache/memcache.go

```diff
@@ -402,24 +402,10 @@ func (c *Client) flushAllFromAddr(addr net.Addr) error {
 // ping sends the version command to the given addr
 func (c *Client) ping(addr net.Addr) error {
 	return c.withAddrRw(addr, func(rw *bufio.ReadWriter) error {
-		if _, err := fmt.Fprintf(rw, "version\r\n"); err != nil {
+		if _, err := fmt.Fprintf(rw, "info clients\r\n"); err != nil {
```

Comment:
Why did you add this commit? I didn't ask you to change this during review.

What was wrong with version? It actually worked. What is "info clients"? I see no reference to that in the protocol or code.

Reply:
I tested again the ping command and I got:
`memcache: unexpected response line from ping: "-ERR unknown command `version`, with args beginning with: \r\n"`

Then I switched to [info](https://redis.io/commands/info) and tested on redis 5 and 4.

Reply:
Ah, sorry, that's what you get when don't sleep enough... I'm reverting it now.